np.sh

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env sh
+
+if test 0 -ne `git status --porcelain | wc -l`; then
```

Comment:
Ahh good point (I just copy pasted from the other pull, since Sindre had said he wanted one), but I can definitely make that change :)